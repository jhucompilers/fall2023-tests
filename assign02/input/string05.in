var s;
s = "A strong smell of petroleum prevails throughout";
println(s);
println(strlen(s));
46;
