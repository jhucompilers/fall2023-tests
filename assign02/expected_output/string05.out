A strong smell of petroleum prevails throughout
47
Result: 46
